src/ui/components/FormOverlay/styles.scss

```diff
@@ -0,0 +1,62 @@
+@import "~core/css/inc/mixins";
+@import "~amo/css/inc/vars";
+@import "~ui/css/vars";
+@import "~photon-colors/colors";
+
+.FormOverlay {
+  align-items: center;
+  background: transparentize($grey-90, 0.5);
+  display: flex;
+  height: 100%;
+  justify-content: center;
+  left: 0;
+  position: fixed;
+  top: 0;
+  width: 100%;
+  z-index: 1000;
+}
+
+.FormOverlay-overlay {
+  background-color: $white;
+  border-radius: $border-radius-s;
+  display: block;
+  margin: 12px;
+  min-width: 55%;
+  padding: 24px;
+}
+
+.FormOverlay-closer {
```

Comment:
Given I'm not sure "closer" is a real noun this read awkwardly to me and I needed to do a double-take. `closeAction`, `closeButton`, etc. might be a bit more descriptive.